src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java

```diff
@@ -72,13 +82,14 @@ public FileResult convert(Param param) throws ConvertException, IOException {
      * @throws IOException when is a invalid file.
      */
     private void generateImage(ImageParam imageParam) throws IOException {
-        commonCommandImage(imageParam);
-
-        commandStrategyList.add(new CommandImageRotate(imageParam.getDegreesToRotate()));
-        commandStrategyList.add(new CommandImageResize(imageParam.getImageWidth(), imageParam.getImageHeight()));
-        commandStrategyList.add(new CommandOutputFilePath(imageParam.getOutputPathFile()));
-        commandStrategyList.add(new CommandOutputFileName(imageParam.getOutputFileName()));
-        commandStrategyList.add(new CommandImageFormat(imageParam.getImageFormat()));
+        commandImageList.add(new CommandImageMagickPath());
+        commandImageList.add(new CommandImageConverter());
+        commandImageList.add(new CommandInputFilePath(imageParam.getInputPathFile()));
+        commandImageList.add(new CommandImageRotate(imageParam.getDegreesToRotate()));
+        commandImageList.add(new CommandImageResize(imageParam.getImageWidth(), imageParam.getImageHeight()));
+        commandImageList.add(new CommandOutputFilePath(imageParam.getOutputPathFile()));
+        commandImageList.add(new CommandOutputFileName(imageParam.getOutputFileName()));
+        commandImageList.add(new CommandImageFormat(imageParam.getImageFormat()));
     }
```

Comment:
That be methods

Reply:
They are in a private method "generateImage"

Reply:
I thing differently, for me add is straight to the list
if I wrong sorry, but add is directly